example/lib/widgets/location_page.dart

```diff
@@ -0,0 +1,208 @@
+import 'package:flutter/material.dart';
+
+import 'package:another_location_plugin/another_location_plugin.dart';
+import '../utils/strings.dart';
+import '../utils/dimensions.dart';
+import '../utils/styles.dart';
+import 'main_button.dart';
+import 'secondary_button.dart';
+
+class LocationPage extends StatefulWidget {
+  const LocationPage({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  _LocationCardState createState() => _LocationCardState();
+}
+
+class _LocationCardState extends State<LocationPage> {
+  bool hasPermissions = false;
+  bool isInitiallized = false;
```

Comment:
this variables are private, you can use InitState to initialize it in false

Reply:
Done!